index.html

```diff
@@ -4998,6 +5000,18 @@ <h2 id="AudioWorkletGlobalScope">
             <code>registerProcessor</code> method is called, but this storage
             is internal and is not directly exposed to the user.
           </p>
+          <div class="note">
+            The <a>AudioWorkletGlobalScope</a> may also contain any other data
+            and code to be shared by these instances. As an example, multiple
+            processors might share an ArrayBuffer defining a wavetable or an
+            impulse response.
+          </div>
+          <div class="note">
+            Every <a>AudioWorkletGlobalScope</a> is associated with at most one
```

Comment:
This actually _is_  the statement of the principle to be enforced. It's what is required to satisfy some of the Worklet motivations described in https://drafts.css-houdini.org/worklets/#motivations, namely that global scopes may be spun up in a rendering environment as needed. Each scope must be able to run consistently, without races.